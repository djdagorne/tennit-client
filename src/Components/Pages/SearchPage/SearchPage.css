.search-segment {
    padding: 30px 15px;
    margin: auto;
    width: 300px;
}
  