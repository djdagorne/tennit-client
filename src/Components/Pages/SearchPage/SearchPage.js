import React, {Component} from 'react';
import {Link} from 'react-router-dom'
import './SearchPage.css';

class SearchPage extends Component {
    render(){
        return(
            <div className="content-container">
                
                <h1 className="banner-text">Search</h1>

                <form className="search">
                <h2 >Find listings in...</h2>

                    <div className="search-segment">
                        <label htmlFor="provence">provence/state:</label>
                        <input type="text" name="provence" placeholder="eg Ontario" /* required *//>
                    </div>

                    <div className="search-segment">
                        <label htmlFor="city">city:</label>
                        <input type="text" name="city" placeholder="eg Toronto" /* required *//>
                    </div>

                    <div className="search-segment">
                        <label htmlFor="price">With monthly rent as high as:</label>
                        <input type="number" name="price" placeholder="eg 750"/>
                    </div>
                    
                    <div className="button-wrap">
                        <Link to="/results">
                            <button type="submit" className="rounded-button">SEARCH</button>
                        </Link>
                    </div>
                </form>
            </div>
        )
    }
}

export default SearchPage;