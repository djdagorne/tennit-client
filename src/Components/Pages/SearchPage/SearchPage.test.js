import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SearchPage from './SearchPage'
import '../../../setupTest'



it('renders without crashing', () => {
        const wrapper = shallow(<SearchPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
})
