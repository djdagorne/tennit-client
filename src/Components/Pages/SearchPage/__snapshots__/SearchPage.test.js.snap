// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="content-container"
>
  <h1
    className="banner-text header-one"
  >
    Search
  </h1>
  <form
    className="search "
    onSubmit={[Function]}
  >
    <h2>
      Find listings in...
    </h2>
    <div
      className="search-segment"
    >
      <label
        htmlFor="province"
      >
        province/state:
      </label>
      <input
        name="searchProvince"
        onChange={[Function]}
        placeholder="eg Ontario"
        type="text"
      />
    </div>
    <div
      className="search-segment"
    >
      <label
        htmlFor="city"
      >
        city:
      </label>
      <input
        name="searchCity"
        onChange={[Function]}
        placeholder="eg Toronto"
        type="text"
      />
    </div>
    <div
      className="search-segment"
    >
      <label
        htmlFor="price"
      >
        With monthly rent as high as:
      </label>
      <input
        name="searchRent"
        onChange={[Function]}
        placeholder="eg 750"
        required={true}
        type="text"
      />
    </div>
    <div
      className="button-wrap"
    >
      <button
        className="rounded-button"
        type="submit"
      >
        SEARCH
      </button>
    </div>
  </form>
</div>
`;
